setup.py

```diff
@@ -42,12 +42,15 @@ def get_packages(package):
 
 env_marker_win = "sys_platform == 'win32'"
 env_marker_below_38 = "python_version < '3.8'"
+env_marker_below_310 = "python_version < '3.10'"
 
 minimal_requirements = [
     "asgiref>=3.4.0",
     "click>=7.0",
     "h11>=0.8",
     "typing-extensions;" + env_marker_below_38,
+    # contextlib2.nullcontext().__aenter__/__aexit__
+    "contextlib2 >= 21.6.0;" + env_marker_below_310,
```

Comment:
Is this really required for 3.9? Is the comment highlighting an issue with the implementation in 3.7, 3.8 & 3.9?

Reply:
on python3.9 the  contextlib.nullcontext is missing `__aenter__` and `__aexit__`